app/src/main/java/com/keylesspalace/tusky/viewdata/NotificationViewData.java

```diff
@@ -15,13 +15,13 @@
 
 package com.keylesspalace.tusky.viewdata;
 
+import androidx.annotation.Nullable;
```

Comment:
why not Java's one?

Reply:
I think it is the standard one we use everywhere else